### PR TITLE
frontend: Art display sort by album and regenerate button 

### LINKED ISSUE #48: frontend: art display refresh button

### LINKED ISSUE #49: frontend: rank by album and not song

### PR DESCRIPTION
**Pull request by ibixler**
fixes #48 fixes #49

this pr adds a regenerate button 
as well as ranking art display by album and not song

### CODE REVIEW

**Review by michibatoryu — COMMENTED** [review]
Tested it out and both of the functionalities work, yay! Is the loading stuff in here for the regenerate button or sort by album functionality? Otherwise maybe we should move that onto another branch for now...

**Review by notsoli — CHANGES_REQUESTED** [review]
ngl this one was rough

**Comment by notsoli** [comment]
Everything works fine, but i'm a little bit flabbergasted that u just overwrote the sort by song code instead of making a new function lol. No need to make an option to toggle yet, but keeping both is a must imo.

File: frontend/src/routes/(app)/+layout.svelte
Comment (notsoli) [inline frontend/src/routes/(app)/+layout.svelte:1]: Why the extra whitespace 😭

**Review by notsoli — CHANGES_REQUESTED** [review]

**Review by notsoli — APPROVED** [review]
pogger

**Pull request merged**